app/utils/ttl-templates/basic-form-template.js

```diff
@@ -8,14 +8,14 @@ export default `@prefix : <#>.
 nodes:24289e48-258f-4919-8c3e-5783a6acb4a4
     a form:Field;
```

Comment:
Maybe for the ttl files change the base to the other branch where you already update this 😅 Otherwise same comments